docs/getting-started/how-to-create-custom-highcharts-files.md

```diff
@@ -95,78 +42,47 @@ export default Highcharts;
 ```
 
 Modify the setup according to your needs, then proceed with the next step.
-Please note that the order of the imports has to match the dependencies of each
-file. Optional additions come therefor last.
+Please note that the order of the imported ES modules should match the
+dependencies of each file. Optional additions come therefor last.
```

Comment:
Typo: "therefore". Also better rewritten: "Optional additions therefore should come last"